### DISCUSSION TITLE
Trino connector not found

### DISCUSSION THREAD

**Discussion by ng22t7a** [Canner/WrenAI#984] [General]
Hi all, I cant find Trino on connector UI. Can someone help me fix this?
![image](https://github.com/user-attachments/assets/23796a44-9626-4fd4-acbe-20b2be111b1c)

**Comment by cyyeh** [comment]
@ng22t7a  Hi, what is your Wren AI version?

**Reply by ng22t7a** [reply]
It's 0.11.0

**Reply by cyyeh** [reply]
@ng22t7a hmm, could you try 0.12.0? https://github.com/Canner/WrenAI/releases/tag/0.12.0
